src/components/AboutPage.js

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import NavigationBar from "./NavigationBar";
+
+export default function AboutPage () {
+    return (
+        <div className="flex flex-col h-screen bg-gray-100 ">
+            
+
+            <div className="flex-1 bg-maroon bg-opacity-80 flex flex-col justify-center items-center">
+                <h1 className='text-5xl text-pink-100 mt-16'>The Novelology Story</h1>
+                <h2 className='text-xl italic text-white mt-10 font-normal'>Turning pages, Uncovering Connections</h2>
+                <h2 className='text-xl italic text-white mt-10 font-normal'> </h2>
+                <h2 className='text-xl italic text-white mt-10 font-normal'> </h2>
+            </div>
+
+
+            <div className="flex-1 bg-gray-100 mt-10">
+                <div className="flex flex-row justify-center items-center w-full">
+                    
+                    <div className="flex flex-row justify-center items-center w-full">
+                        <div className="max-w-3xl">
+                            <div className="text-left mb-14">
+                                <h2 className="text-2xl font-normal text-maroon">About</h2>
+                                <p className="font-normal mt-2 text-xl text-justify">At Novelology, we're passionate about connecting avid readers through our social media book platform. Our platform offers numerous features designed to enhance your reading experience and foster lively discussions within the book community. Our inspiration stemmed from a desire to create a space where the book community could thrive.</p>
```

Comment:
Can we link or talk more about the numerous features that we offer that another platform wouldn't?

Reply:
resolved it, i added a link to the home page per Rishitha